spec/scenic/adapters/postgres_spec.rb

```diff
@@ -51,16 +60,46 @@ module Adapters
 
           expect(adapter.views.map(&:name)).not_to include("greetings")
         end
+
+        it "raises an exception if the version of PostgreSQL is too old" do
+          adapter = Postgres.new
+          allow(ActiveRecord::Base.connection)
+            .to receive(:postgresql_version) { 90202 }
+
+          expect { adapter.drop_materialized_view("greetings") }
+            .to raise_error Scenic::Postgres::MaterializedViewsNotSupportedError
+        end
       end
 
       describe "#refresh_materialized_view" do
-        it "raises descriptive error if concurrent refresh is not possible" do
+        it "raises an exception if the version of PostgreSQL is too old" do
           adapter = Postgres.new
-          adapter.create_materialized_view(:tests, "SELECT text 'hi' as text")
+          allow(ActiveRecord::Base.connection)
+            .to receive(:postgresql_version) { 90202 }
+
+          expect { adapter.refresh_materialized_view(:tests) }
+            .to raise_error Scenic::Postgres::MaterializedViewsNotSupportedError
+        end
 
-          expect {
-            adapter.refresh_materialized_view(:tests, concurrently: true)
-          }.to raise_error(/Create a unique index with no WHERE clause/)
+        context "refreshing concurrently" do
+          it "raises descriptive error if concurrent refresh is not possible" do
+            adapter = Postgres.new
+            adapter.create_materialized_view(:tests, "SELECT text 'hi' as text")
+
+            expect { adapter.refresh_materialized_view(:tests, concurrently: true) }
+              .to raise_error(/Create a unique index with no WHERE clause/)
+          end
+
+          it "raises an exception if the version of PostgreSQL is too old" do
+            adapter = Postgres.new
+            allow(ActiveRecord::Base.connection)
```

Comment:
similar feedback to the above here - let's pass in a double instead, and have it respond to the correct messages the way we want. Then let's unit test `supports_concurrent_refreshes` in the connection decorator specs.